sonar/modules/users/cli.py

```diff
@@ -67,9 +67,15 @@ def import_users(infile):
             password = hash_password(password)
             del user_data['password']
 
+            if not user_data.get('roles'):
+                user_data['roles'] = [UserRecord.ROLE_USER]
```

Comment:
Why not having the roles in the fixtures file?

Reply:
Normally it's the case, it's just to insure that the user as at least the role "user".